### ISSUE TITLE
The PDBQT files appear distorted after being converted from the SDF format!

### ISSUE THREAD

**Issue by nilanjana1996** [openbabel/openbabel#2720]
Hi @fredrikw,
As you suggested, I've closed the previous issue and am now creating a new one concerning the "distorted" PDBQT files.

The issue occurs when I visualize the PDBQT files in PyMol. Some structures appear distorted in PyMol (as shown in the attached images). 

Consequently, when I use AutoDock Vina to dock the molecule against a specific receptor and then try to visualize the output file in Discovery Studio for ligand-protein interaction, the software cannot recognize the ligand as a single entity. Instead, it identifies two separate fragments, preventing it from generating a 2D interaction diagram.

I've attached a few screenshots for reference.

The PDBQT files were generated by splitting a single large SDF file into separate fragments, energy minimizing them, and then converting them into PDBQT files using OpenBabel.

 
![Screenshot 2024-08-27 134607](https://github.com/user-attachments/assets/11c20fd4-d646-4b8d-938a-e68ea6282f49)
![Screenshot 2024-08-27 134733](https://github.com/user-attachments/assets/02cc1023-27f0-4012-950c-10c77796bdbf)

**Comment by fredrikw** [comment]
Could you please also attach the corresponding sdf- and pdbqt-files?

**Comment by nilanjana1996** [comment]
Yes.

I tried with a subset of the large SDF file containing only 10 ligands, and processed them as I explained it last comment. 
The files can be accessed in the Google drive link, [](https://drive.google.com/drive/folders/19tDdzlWhIfmRpBGNMW1mINjm9RF0LBR3?usp=drive_link)

I have also uploaded docking output from autodock vina of two ligand, emem8 and emem9 respectively.

**Comment by nilanjana1996** [comment]
Hi @fredrikw 

It seems the issue has been resolved.

This time, I performed energy minimization on the combined SDF file and then split it into separate SDF files in next step. 

Subsequently, I used the "--gen3D" flag while converting the individual SDF files to PDBQT, which successfully generated the 3D structures.

Therefore, I believe the problem is now solved, and I'm closing the issue.

Thank you for your assistance.
